modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/MultiSelect.js

```diff
@@ -51,7 +51,9 @@ export default function MultiSelect({
 					<ClayMultiSelect
 						className={cssClass}
 						clearAllTitle={clearAllTitle}
-						closeButtonAriaLabel={closeButtonAriaLabel}
+						closeButtonAriaLabel={Liferay.Language.get(
```

Comment:
This value is already being passed localized from the tag, we don't need to look for the key again (at this point the value is `Remove {0}`, not the key `remove-x`)